pom.xml

```diff
@@ -528,7 +528,7 @@ under the License.
 				<artifactId>jackson-bom</artifactId>
 				<type>pom</type>
 				<scope>import</scope>
-				<version>2.12.1</version>
+				<version>2.13.2.20220328</version>
```

Comment:
This contains the Jackson Databind 2.13.2.2 and has the fix for CVE-2020-36518,  `https://github.com/FasterXML/jackson-databind/commits/jackson-databind-2.13.2.2`